jenkins_pipelines/manager_prs/Jenkinsfile_prs_backend_unitpsql

```diff
@@ -28,15 +28,22 @@ pipeline {
    }
    // run only on specific hosts
     agent { label 'suse-manager-unit-tests' }
-   // trigger
+    
+    parameters {
+        booleanParam(defaultValue: true, description: 'Clean up workspace after a successful execution.', name: 'cleanWorkspace')
+    }
+    
     triggers {
         cron('H/5 * * * *')
-       }
+    }
+    
     stages {
         stage('Clean Up Workspace') {
             steps {
                 echo 'Clean up previous workspace'
                 cleanWs()
+                echo 'Check out SCM'
```

Comment:
FMPOV, checking out should be a second step.

Reply:
So, the checkout is done at the beginning of the job. But as we do a cleanWs, this checkout is deleted and to fix it, we checkout again here. But it gonna be ugly in the pipeline to have two times the checkout stage, it's way is a second step in the same stage.

Reply:
Ah, right, damned pipelines...